asmcli/asmcli

```diff
@@ -1990,11 +1992,13 @@ init_meshconfig_managed() {
 
   info "Initializing meshconfig managed API..."
   local POST_DATA
-  POST_DATA='{"workloadIdentityPools":["'${PROJECT_ID}'.hub.id.goog","'${PROJECT_ID}'.svc.id.goog"], "prepare_istiod": true}'
-  init_meshconfig_curl "${POST_DATA}" "${PROJECT_ID}"
+  # Initialize the Hub Hosting project with Hub WIP
+  POST_DATA='{"workloadIdentityPools":["'${FLEET_ID}'.hub.id.goog","'${FLEET_ID}'.svc.id.goog"], "prepare_istiod": true}'
```

Comment:
I think this is actually switched (we don't need to prepare istiod's for the fleet project, but we do for the cluster project).